cli/src/main/java/com/devonfw/tools/ide/cli/Ideasy.java

```diff
@@ -59,9 +59,13 @@ public int run(String... args) {
       exitStatus = runOrThrow(args);
     } catch (CliException error) {
       exitStatus = error.getExitCode();
-      String errorMessage = error.getMessage();
-      if ((errorMessage != null) && !errorMessage.isBlank()) {
-        context().error(errorMessage);
+      if (context().debug().isEnabled()) {
+        context().error(error);
+      } else {
+        String errorMessage = error.getMessage();
+        if ((errorMessage != null) && !errorMessage.isBlank()) {
+          context().error(errorMessage);
+        }
```

Comment:
This is not required for the fix.
I would propose to revert it from here and create a new ticket and PR for the general feature that when debug is enabled we also get stacktraces of CliExceptions.
```suggestion
      String errorMessage = error.getMessage();
      if ((errorMessage != null) && !errorMessage.isBlank()) {
        context().error(errorMessage);
```